src/app/task-form-dialog/task-form-dialog.component.ts

```diff
@@ -53,4 +53,17 @@ export class TaskFormDialogComponent implements OnInit {
       )
     }
   }
+  titleValidator(control:FormControl)
```

Comment:
Why is it better to have a factory function that returns a function (validator itself) when creating custom validator, as stated in the documentation? Because in that case you'll be able to pass extra params to the validator. Let's say you need to create a validator function to check if the name of something is unique. You will have to pass an array of already existing instances to check their names